src/RedisEnterpriseCache/examples/Invoke-AzRedisEnterpriseCacheForceDatabaseUnlink.md

```diff
@@ -0,0 +1,6 @@
+### Example 1: Unlink database
+```powershell
+PS C:\> Invoke-AzRedisEnterpriseCacheForceDatabaseUnlink -ResourceGroupName "MyGroup" -ClusterName "MyCache3" -Id @("<database-id>")
```

Comment:
```suggestion
Invoke-AzRedisEnterpriseCacheForceDatabaseUnlink -ResourceGroupName "MyGroup" -ClusterName "MyCache3" -Id @("<database-id>")
```
Please remove PS C:> from all examples to avoid breaking user copy-paste-use experience.
BTW, for the command without output, please use following placeholder to present none output
\```output 
None 
\```